test/Dialect/FIRRTL/emit-metadata.mlir

```diff
@@ -131,7 +131,7 @@ firrtl.circuit "top"
     firrtl.module @Mem2() {
       %head_MPORT_3 = firrtl.mem Undefined  {depth = 20 : i64, name = "head", portNames = ["MPORT_2", "MPORT_6"], readLatency = 1 : i32, writeLatency = 1 : i32} : !firrtl.bundle<addr: uint<5>, en: uint<1>, clk: clock, data: uint<5>, mask: uint<1>>
     }
-    firrtl.module @dut() attributes {annotations = [
+    firrtl.module @DUT() attributes {annotations = [
```

Comment:
Note that this change wasn't necessary, but I found it a bit easier to read when module names have a capital initial letter and instance names do not. It also helped me sanity check that my change worked correctly by distinguishing between the DUT's module name and its instance name from within the `top` module.